plugins/authentication/src/OAuthModel/model.tsx

```diff
@@ -318,24 +334,37 @@ const stateModelFactory = (configSchema: OAuthInternetAccountConfigModel) => {
           token: string,
           location: UriLocation,
         ): Promise<string> {
-          const decoded = jwtDecode<JwtPayload>(token)
```

Comment:
is the jwt decode unneeded now? I wasn't sure what it was for initially. also, is this PR preferable to https://github.com/GMOD/jbrowse-components/pull/3466  or can https://github.com/GMOD/jbrowse-components/pull/3466 be added after the changes that were made?

Reply:
This supersedes #3466. It includes the commits in that PR, with additional work after that.

As for the JWT decode, this code was attempting to find the expiration time from the token. However, it only worked if the token was a JWT token constructed in a particular way, which the OAuth spec does not require, and I'm not even sure how common it is, so it was removed for better compatibility.